examples/md-flexible/src/Thermostat.h

```diff
@@ -155,9 +157,9 @@ void addBrownianMotion(AutoPasTemplate &autopas, ParticlePropertiesLibraryTempla
     // we need one random engine and distribution per thread
     std::default_random_engine randomEngine(42 + autopas::autopas_get_thread_num());
     std::normal_distribution<double> normalDistribution{0, 1};
-    for (auto iter = autopas.begin(); iter.isValid(); ++iter) {
-      addMaxwellBoltzmannDistributedVelocity(*iter, factors[iter->getTypeId()], randomEngine, normalDistribution);
-    }
+    autopas.forEach([&](auto &particle) {
```

Comment:
I do actually prefer this way of replacing the for loops, i.e., directly specify the lambda within the forEach function.

Reply:
Do you feel this way also if the lambda function is more than a one-liner?
If so I can rewrite the others.

Reply:
No longer relevant for this isse, but for completeness: lambdas will be inlined in example code.